src/core/cgroup.c

```diff
@@ -506,6 +515,23 @@ static unsigned cgroup_apply_blkio_device_limit(const char *path, const char *de
         return n;
 }
 
+static bool cgroup_context_has_unified_memory_config(CGroupContext *c) {
+        return c->memory_low || c->memory_high != CGROUP_LIMIT_MAX || c->memory_max != CGROUP_LIMIT_MAX;
```

Comment:
Please use `c->memory_low > 0` instead. We generally only use `bool`s and pointers directly as booleans.